services/azure-devops/azure-devops-build.tester.js

```diff
@@ -20,6 +20,10 @@ t.create('named branch')
     message: isBuildStatus,
   })
 
+t.create('never built definition')
+  .get('/swellaby/opensource/112.json')
+  .expectBadge({ label: 'build', message: 'never built' })
```

Comment:
I figured it'd be worth adding a test for this scenario, but can certainly remove it if folks feel strongly otherwise. I've got the target build pipeline here locked so that it will never run, keeping this test passing.